poki-sdk/manifests/web/engine_template.html

```diff
@@ -19,7 +19,7 @@
 		Module['onRuntimeInitialized'] = function() {
 			PokiSDK.init().then(()=>{
 				Module.runApp("canvas");
-			}).catch(()=>{
+			}, ()=>{
```

Comment:
Added `reject` handler. It's called in case of reject or exception in Promise. If exception happens in `resolve` handler it will be passed outside of promise and handled by `window` error handler.